app/screens/WelcomeScreen.tsx

```diff
@@ -1,35 +1,48 @@
 import { observer } from "mobx-react-lite"
 import React from "react"
-import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import { Dimensions, Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
 import { SafeAreaView } from "react-native-safe-area-context"
-import { Text } from "../components"
-import { isRTL } from "../i18n"
+import { Button, Screen, Text } from "../components"
+import { AppStackScreenProps } from "../navigators"
 import { colors, spacing } from "../theme"
 
-const welcomeLogo = require("../../assets/images/logo.png")
-const welcomeFace = require("../../assets/images/welcome-face.png")
+const welcomeLogo = require("../../assets/images/welcome-shapes.png")
+
+interface WelcomeScreenProps extends AppStackScreenProps<"Welcome"> {}
+
+export const WelcomeScreen: React.FC<WelcomeScreenProps> = observer(function WelcomeScreen(_props) {
+  const { navigation } = _props
```

Comment:
This is an Ignite issue, but there's an inconsistency with how we pull in navigation, since we use the hook for the generated screens, but here we use the prop. My preference is the hook, since it's available even in components down the tree which don't have the prop available, and then it can be consistent throughout.